server/views/pages/contacts/manage/prisonerSearchResults.njk

```diff
@@ -83,7 +83,7 @@
             {% set rows = [] %}
             {% for result in results.content %}
                 {% set prisonerNameHtml %}
-                   <a href="/prisoner/{{ result.prisonerNumber }}/contacts/list/{{journey.id}}" class="govuk-link govuk-link--no-visited-state bapv-result-row">
+                   <a href="/prisoner/{{ result.prisonerNumber }}/contacts/list/{{journey.id}}" class="govuk-link govuk-link--no-visited-state bapv-result-row" data-qa="prisoner-name">
```

Comment:
should maybe be indexed or include the prisoner number in the data-qa to make sure you select the right one

Reply:
prisoner number in the data-qa added